ts/data/sample.ts

```diff
@@ -858,31 +912,47 @@ function addPhixContents(
     metrics.forEach((metric) => addMetricRequirementText(metric, fragment));
   };
 
-  const multipleLanes = sample.run.lanes.length > 1;
-  sample.run.lanes.forEach((lane) => {
+  const processLane = (lane: Lane, multipleLanes: boolean) => {
     const laneDiv = document.createElement("div");
     laneDiv.classList.add("whitespace-nowrap", "print-hanging");
-    let text = multipleLanes ? `L${lane.laneNumber}` : "";
+
+    let text = multipleLanes ? `L${lane.laneNumber}: ` : "";
     if (nullOrUndefined(lane.percentPfixRead1)) {
-      const span = document.createElement("span");
-      span.innerText = text + ": ";
-      laneDiv.appendChild(span);
-      laneDiv.appendChild(makeNotFoundIcon());
+      text += `<span>${makeNotFoundIcon()}</span>`;
     } else {
-      if (multipleLanes) {
-        text += " ";
-      }
       text += `R1: ${lane.percentPfixRead1}`;
       if (!nullOrUndefined(lane.percentPfixRead2)) {
         text += `; R2: ${lane.percentPfixRead2}`;
       }
-      laneDiv.innerText = text;
-      if (addTooltip) {
-        tooltip.addTarget(laneDiv, addContents);
-      }
     }
-    fragment.appendChild(laneDiv);
+
+    laneDiv.innerHTML = text;
```

Comment:
This is still going to result in the text "[object HTMLElement]" if values are missing because `makeNotFoundIcon` returns an object that is meant to be appended to the div as it originally was. `innerHtml` should not be used here, and is best avoided in most cases since it is more error-prone (e.g. here), and will perform poorer than creating and appending elements. Again, there was no reason to change this code which was previously working fine.